### ISSUE TITLE
Logstash Prefer a Local copy of ElasticSearch

### ISSUE THREAD

**Issue by edsealing** [sealingtech/EDCOP-TOOLS#6]
Currently, Logstash is setup to point to the "elasticsearch" service. This service load-balances the events to all of the available elasticsearch nodes. Preferring a local copy of elasticsearch (on the same node) would likely increase performance within the cluster and allow for better scalability.

Kubernetes has a ticket in for this already: https://github.com/kubernetes/kubernetes/issues/7433

Potential workaround:
- Use a "NodePort" or hostport in Elasticsearch and point the LogStash to localhost:9200

**Comment by dlohin** [comment]
@edsealing I don't think this is a good idea for a variety of reasons.  I know you are attempting to keep network traffic down but it harms a lot of flexibility.
